UI/obs-app.cpp

```diff
@@ -1098,10 +1098,75 @@ void OBSApp::ParseExtraThemeData(const char *path)
 	setPalette(pal);
 }
 
-bool OBSApp::SetTheme(std::string name, std::string path)
+bool OBSApp::ParseThemeMeta(const char *path)
 {
-	theme = name;
+	BPtr<char> data = os_quick_read_utf8_file(path);
+	CFParser cfp;
+	int ret;
+
+	cf_parser_parse(cfp, data, path);
+
+	themeMeta = nullptr;
+
+	if (cf_go_to_token(cfp, "OBSThemeMeta", nullptr)) {
```

Comment:
Add `cf_token_is` test to avoid `cf_go_to_token` skipping first token.

Reply:
Done. Thanks for the suggestion.